packages/core/admin/admin/src/pages/ProfilePage/index.js

```diff
@@ -444,7 +448,7 @@ const ProfilePage = () => {
                               },
                               {
                                 documentation: (
-                                  <a
+                                  <DocumentationLink
```

Comment:
I know but I think it is because it adds a "external link" icon to the link, and here we do not want it for some reason

Reply:
@maevalienard would it be ok if we had icons on the links like so?
![image](https://user-images.githubusercontent.com/89356961/155999380-777563b0-56eb-437a-9079-99c73bda20d7.png)

Reply:
It depends where you add the icons, because for instance in the User profile page, the link is in the perfect middle of a sentence, so that wouldn't make much sense to have the icon in the middle of the sentence as well

Reply:
Get it, @soupette wdyt?